conference/react-js/src/reducers/talks.js

```diff
@@ -6,7 +6,7 @@ const talksInitial = [];
 
 const talks = (state = talksInitial, action) => {
   if (action.type === LOAD) {
-    return [...state, ...action.payload];
+    return [...action.payload];
```

Comment:
seems like you can return `action.payload` by itself. I see no sense of using `spread` operator along with square brackets